data/projects/tlsn.ts

```diff
@@ -17,9 +21,19 @@ export const tlsn: ProjectInterface = {
     discord: "https://discord.gg/9XwESXtcN7",
   },
   tags: {
-    themes: ["build"],
-    type: [],
-    builtWith: [],
-    keywords: [],
+    themes: ["build", "play"],
+    type: [
```

Comment:
```suggestion
    types: [
```
please consider to add this change that will be required after this PR: https://github.com/privacy-scaling-explorations/website-v2/pull/87